src/Microsoft.SqlTools.ServiceLayer/Connection/Contracts/LanguageFlavorChange.cs

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.Connection.Contracts
+{
+
+    /// <summary>
+    /// Parameters for the Language Flavor Change notification.
+    /// </summary>
+    public class LanguageFlavorChangeParams
+    {
+        /// <summary>
+        /// A URI identifying the affected resource         
+        /// </summary>
+        public string Uri { get; set; }
+
+        /// <summary>
+        /// The primary language
+        /// </summary>
+        public string Language { get; set; }
```

Comment:
I would prefer not to assume this. I would like to keep this generic enough that others could use in the future (even though for now, this is the most likely option)